vkconfig/vuidsearchwidget.cpp

```diff
@@ -19,72 +19,80 @@
  * - Christophe Riccio <christophe@lunarg.com>
  */
 
+#include <QTimer>
+
 #include "vuidsearchwidget.h"
 #include "vk_vuids.h"
 
 VUIDSearchWidget::VUIDSearchWidget(QWidget *parent) : QWidget(parent) {
     int nNumElements = sizeof(vuids) / sizeof(vuids[0]);
-    for (int i = 0; i < nNumElements; i++) vuid_list_ << vuids[i];
+    for (int i = 0; i < nNumElements; i++) _vuid_list << vuids[i];
 
     // Completer does not need this sorted, but we do
-    vuid_list_.sort();
+    _vuid_list.sort();
 
-    user_box_ = new QLineEdit(this);
-    user_box_->setFocusPolicy(Qt::StrongFocus);
+    _user_box = new QLineEdit(this);
+    _user_box->setFocusPolicy(Qt::StrongFocus);
 
-    add_button_ = new QPushButton(this);
-    add_button_->setText("Add");
+    _add_button = new QPushButton(this);
+    _add_button->setText("Add");
 
-    user_box_->show();
-    user_box_->setText("");
-    user_box_->installEventFilter(this);
+    _user_box->show();
+    _user_box->setText("");
+    _user_box->installEventFilter(this);
 
-    search_vuid_ = nullptr;  // Safe to delete a pointer
+    _search_vuid = nullptr;  // Safe to delete a pointer
     ResetCompleter();
 
-    connect(add_button_, SIGNAL(pressed()), this, SLOT(addButtonPressed()));
-    connect(user_box_, SIGNAL(returnPressed()), this, SLOT(addButtonPressed()));
+    connect(_add_button, SIGNAL(pressed()), this, SLOT(addButtonPressed()));
+    // connect(_user_box, SIGNAL(returnPressed()), this, SLOT(addButtonPressed()));
```

Comment:
Is this an intentional removal?

If so, do we need to keep the commented-out code?

Reply:
Removed for testing. made permanent... forgot to delete. Another good catch, and further justification for not cramming too much into one PR.